gallery/src/pages/Accessibility.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+import { NxP, NxCode, NxH3, NxList, NxTextLink } from '@sonatype/react-shared-components';
+
+const Accessibility = () =>
+  <GalleryTile title="Accessibility">
+    <NxP>
+      Accessibility (A11y) means ensuring that our applications are usable by everybody. We do this in a variety of
+      different ways. We make sure that our designs are clear and that they follow accessibility guidelines for
+      contrast and readability. Our components are written with semantic HTML, correct use of <NxCode>aria</NxCode>
+      tags, and testing in VoiceOver and ChromeVox accessibility is built into the Shared Components. However there is
+      still work that needs to be done to make sure that the application as a whole is accessible.
+    </NxP>
+
+    <NxH3 className="nx-tile__section-header">Making your pages accessible</NxH3>
+    <NxP>
+      Ensuring you use the components in an accessible manner is important, here are a few tips to help you make your
+      pages accessible.
+    </NxP>
+
+    <NxList className="nx-list--bulleted">
+      <NxList.Item>Tab order is important. Make sure that the tab order of the page is logical.</NxList.Item>
+      <NxList.Item>Related to the above, does the keyboard focus order follow the visual layout?</NxList.Item>
+      <NxList.Item>Unique title element for each page or view?</NxList.Item>
+      <NxList.Item>Do all images and visual graphs have alt text?</NxList.Item>
+      <NxList.Item>
+        Make sure that you take advantage of the props and attributes that have been added to the components to improve
+        their accessibility. Many component documentation pages have an Accessibility section with extra information.
+      </NxList.Item>
+      <NxList.Item>
+        <NxTextLink external href="https://developer.mozilla.org/en-US/docs/Glossary/Semantics">
+          Semantic HTML
+        </NxTextLink>
+        {' '}is a critical aspect of accessibility. Headings should always use the correct
+        {' '}<NxCode>&lt;h#&gt;</NxCode> tag, paragraphs should be wrapped in <NxCode>&lt;p&gt;</NxCode> tags, etc.
+      </NxList.Item>
+      <NxList.Item>
+        All form elements should have a label.
```

Comment:
Depending on how this is interpreted, it isn't necessarily true.  For instance, a filter input in a table doesn't have a label.  It should still have an accessible name, and can be given one via a placeholder or `aria-label`.  It also isn't necessarily a "form element", depending on how the reader defines that term (it isn't in a form).  Maybe say "all inputs within forms" or something to be more clear.  Also mention which RSC components create labels (`NxFormGroup`)

Reply:
> Depending on how this is interpreted, it isn't necessarily true. For instance, a filter input in a table doesn't have a label. It should still have an accessible name, and can be given one via a placeholder or `aria-label`. It also isn't necessarily a "form element", depending on how the reader defines that term (it isn't in a form). Maybe say "all inputs within forms" or something to be more clear. Also mention which RSC components create labels (`NxFormGroup`)

https://github.com/sonatype/sonatype-react-shared-components/pull/455/commits/98f1ba361fa1ce35c20b1f60685045e2d7d8f569